src/modal/modal.module.ts

```diff
@@ -17,7 +17,7 @@ import { PlaceholderModule } from "./../placeholder/placeholder.module";
 import { ExperimentalModule } from "./../experimental.module";
 
 // exports
-export { default as Modal } from "./modal.decorator";
+export { default as ModalDecorator } from "./modal.decorator";
```

Comment:
Wouldn't this be a breaking change for everybody importing `Modal`?

Reply:
As far as I can tell it already was ... this just corrects the exports to match what the code was doing (there were two `Modal` exports, so I _think_ the named one takes precedence ... there's no way to `import { Modal }` and get the decorator right now)

Reply:
It also won't compile unless we remove the duplicate export names, so that's a thing 😱